tensorflow/python/kernel_tests/embedding_ops_test.py

```diff
@@ -748,6 +748,29 @@ def testEmbeddingLookupSparse(self):
         atol = rtol
         self.assertAllClose(np_embedding_sum, tf_embedding_sum, rtol, atol)
 
+  def testMissingInSparseIds(self):
+    # Github issue, 36359
+    with self.test_session():
+      x = array_ops.ones((4, 5))
+      sp_ids = sparse_tensor.SparseTensor(
+          constant_op.constant([[1, 0], [3, 0]], dtypes.int64),
+          constant_op.constant([0, 2], dtypes.int32),
+          constant_op.constant([4, 1], dtypes.int64))
+      sp_weights = sparse_tensor.SparseTensor(
+          constant_op.constant([[1, 0], [3, 0]], dtypes.int64),
+          constant_op.constant([1, 1], dtypes.float32),
+          constant_op.constant([4, 1], dtypes.int64))
+
+      for combiner in ["sum", "mean", "sqrtn"]:
+        embedding_sum = embedding_ops.embedding_lookup_sparse(
+            x, sp_ids, sp_weights, combiner=combiner)
+
+        tf_embedding_sum = ops.convert_to_tensor(embedding_sum).numpy()
```

Comment:
@ashutosh1919 can you please check this error 

`AttributeError: 'Tensor' object has no attribute 'numpy'
`